client/src/components/ReportSummary.js

```diff
@@ -15,6 +15,7 @@ import PropTypes from "prop-types"
 import React, { useEffect, useRef, useState } from "react"
 import { Col, Grid, Label, Row } from "react-bootstrap"
 import utils from "utils"
+import PlanningConflictForReport from "components/PlanningConflictForReport"
```

Comment:
Please properly order the imports (alphabetically on imported package if there's no command for it in your IDE).